schema/fixtures.js

```diff
@@ -292,6 +292,38 @@
               ]
             }
           ]
+        },
+        {
+          "id": 10,
+          "user": { "id": 5 },
+          "created_at": "2001-06-01T01:00:00",
+          "private_location": "1,2",
+          "project_ids": [ 1 ],
+          "private_geojson": { "type": "Point", "coordinates": [ "2", "1" ] },
+          "project_observations": [
+            {
+              "user_id": 1,
+              "uuid": "8cf0282d-c6f3-4947-9905-4937def371f4",
+              "project_id": 1,
+              "curator_coordinate_access": true
```

Comment:
Actually - I think this needs to happen before merging, because the API returns the preference as `allows_curator_coordinate_access`, and we need to maintain that label. So I'd recommend fixing the fixtures, changing the name of the preference in `viewerHasCuratorCoordinateAccess` and making sure the tests still pass